rabies/analysis_pkg/diagnosis_pkg/analysis_QC.py

```
@@ -433,10 +411,13 @@ def QC_distributions(prior_map,FC_maps,network_var,DR_conf_corr, total_CRsd, mea
 
     # apply QC thresholds
     QC_inclusion = np.ones(len(scan_name_list)).astype(bool)
+    plot_QC_thresholds = {}
     if not scan_QC_thresholds['Dice'] is None:
```

Ruff (0.15.6)

[error] Test for object identity should be `is not`

Convert to `is not`

(E714)